### PR TITLE
Clarification to user role assignment

### PR DESCRIPTION
**Pull request by jnummelin**
This PR clarifies the docs a bit around user role assignment.

### CODE REVIEW

**Comment by jakolehm** [comment]
LGTM

File: docs/using-kontena/users.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Kontena has built-in role based user management. The first user that logs in to
 
 **user**
   * can only operate within grids
+  > User role is automatically assigned to any user added to a grid using the [Add User to Grid](users#add-users-to-role) command and there's no need to specifically add it.
```
Comment (nevalla) [inline docs/using-kontena/users.md]: should the link point to `users#add-users-to-grid`?

**Comment by jnummelin** [comment]
Links fixed

**Comment by nevalla** [comment]
:+1:

**Pull request merged**